templates/commands/render_action_regexreplace_test.go

```diff
@@ -52,6 +52,42 @@ func TestActionRegexReplace(t *testing.T) {
 				"a.txt": "alpha bar gamma",
 			},
 		},
+		{
+			name: "default_multiline_false",
+			initContents: map[string]string{
+				"a.txt": "apple banana\nbanana apple\napple apple\n",
+			},
+			rr: &model.RegexReplace{
+				Paths: modelStrings([]string{"."}),
+				Replacements: []*model.RegexReplaceEntry{
+					{
+						Regex: model.String{Val: "\n$"},
+						With:  model.String{Val: ""},
+					},
+				},
+			},
+			want: map[string]string{
+				"a.txt": "apple banana\nbanana apple\napple apple",
+			},
+		},
+		{
+			name: "override_multiline_true",
+			initContents: map[string]string{
+				"a.txt": "apple banana\nbanana apple\napple apple\n",
+			},
+			rr: &model.RegexReplace{
+				Paths: modelStrings([]string{"."}),
+				Replacements: []*model.RegexReplaceEntry{
+					{
+						Regex: model.String{Val: "(?flags:re m)\n$"},
```

Comment:
Hmm, this was working before when I had `(?flags:re m)^apple` I wonder why its saying regex not valid now. Was trying to get a more compelling example, but it seems no good deed goes unpunished.

Reply:
You'd do `(?m:your_regex)`.

Reply:
Thanks!